internal/characters/wanderer/dash.go

```diff
@@ -23,18 +23,10 @@ func init() {
 }
 
 func (c *char) Dash(p map[string]int) action.ActionInfo {
-	delay := c.checkForSkillEnd()
-
 	if c.StatusIsActive(skillKey) {
 		return c.WindfavoredDash(p)
 	}
-
-	ai := c.Character.Dash(p)
-	ai.Frames = func(action action.Action) int { return delay + ai.Frames(action) }
-	ai.AnimationLength = delay + ai.AnimationLength
-	ai.CanQueueAfter = delay + ai.CanQueueAfter
-
-	return ai
```

Comment:
In existing implementation, `c.checkForSkillEnd()` will always return 0 if `c.StatusIsActive(skillKey)`. The modifying `ai` is useless since delay will always be 0 in these cases

`c.WindfavoredDash` will not run the dash CD logic as desired

Reply:
You have to at least keep the `c.checkForSkillEnd()` call in order to have the E state end properly here.

Reply:
It's possible that `c.StatusIsActive(skillKey)` is true, but `c.skydwellerPoints` are < 0. In this case the normal dash should be extended by `delay` (Wanderer falls down and only then dashes). So please revert this change.